examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js

```diff
@@ -0,0 +1,22 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `YouTube Playback`
+    .page `https://www.youtube.com/watch?v=I6WuCe1My9w`
+    .meta({ autoplay: true });
```

Comment:
Why does the `meta` call is necessary (here and below)?

Reply:
Looks nicer in the console IMHO:
https://github.com/DevExpress/testcafe-examples/pull/22/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R18